embedly/embedly/api/views.py

```diff
@@ -34,8 +34,8 @@ def version():
     )
 
 
-@blueprint.route('/extract')
-def extract_urls():
+@blueprint.route('/extract', methods=['GET'])
+def extract_urls_v1():
```

Comment:
OK. Had some more thoughts about it. Here's how I propose we do things:

We leave V1 as it is, push an update to the addon that does V2 with a V1 fallback.
Some days later, we turn off V1.

How does that sound?